server/controllers/profile.js

```diff
@@ -22,5 +39,37 @@ exports.savePhoto = asyncHandler(async (req, res, next) => {
     fs.unlinkSync(photo.path);
   });
 
-  res.status(200).json(urls);
-});
\ No newline at end of file
+  const profile = await Profile.findOneAndUpdate({userId: user.id}, {profilePic: urls[0]});
+  
+  if (profile) {
+    const oldUrl = profile.profilePic;
```

Comment:
Why don't we perform a check to see if a user has a `profilePic` and only then delete it?

Reply:
@DejanKatovic Please address comment above